runtime/doc/pack.txt

```diff
@@ -416,13 +417,16 @@ add({specs}, {opts})                                          *vim.pack.add()*
                  • {confirm}? (`boolean`) Whether to ask user to confirm
                    initial install. Default `true`.
 
-del({names})                                                  *vim.pack.del()*
+del({names}, {opts})                                          *vim.pack.del()*
     Remove plugins from disk
 
     Parameters: ~
       • {names}  (`string[]`) List of plugin names to remove from disk. Must
                  be managed by |vim.pack|, not necessarily already added to
                  current session.
+      • {opts}   (`table?`) A table with the following fields:
+                 • {force}? (`boolean`) Whether to allow deleting an active
+                   plugin. Default `false`.
```

Comment:
I would probably not bother with this option. Showing an error is enough, then the user can fix the situation.